source/libs/on-new-comments.ts

```diff
@@ -38,8 +38,10 @@ function addListeners(): void {
 		childList: true
 	});
 
-	// When hidden comments are loaded by clicking "Load more..."
-	delegates.add(delegate('.js-ajax-pagination', 'page:loaded', run));
+	// When hidden comments are loaded by clicking "Load more…"
+	delegates.add(delegate('.js-ajax-pagination', 'submit', event => {
+		event.target?.addEventListener('page:loaded', run, {once: true});
+	}));
```

Comment:
Never generate a new function here because `delegate` won't deduplicate it.

Also, what does this mean? `event.target` is still `.js-ajax-pagination`, so why listen to `submit` first and then add a separate `page:loaded` listener later?

Reply:
When clicking the "Load more…" button, the form is submitted and the XHR/pjax request is started. When the response arrives, the form is replaced with the new contents from the server (possibly containing another "Load more…" form. After that, the `page:loaded` event is triggered, but at that time the form is not part of the DOM tree anymore, i.e. it doesn't have a `parentNode`, so the event can't bubble up to the `document` for `delegate` to catch it.